biz.aQute.bnd.gradle/README.md

```diff
@@ -363,7 +371,7 @@ tasks.register('export', Export) {
 }
 ```
 
-There are five properties which can be configured for an Export task:
+There are six properties which can be configured for an Export task:
```

Comment:
Same comment as above (avoid a concrete number in the description to reduce diff).